src/main/java/com/mixpanel/android/mpmetrics/AnalyticsMessages.java

```diff
@@ -130,8 +131,8 @@ protected MPConfig getConfig(Context context) {
         return MPConfig.getInstance(context);
     }
 
-    protected ServerMessage getPoster() {
-        return new ServerMessage();
+    protected RemoteService getPoster() {
```

Comment:
what's the difference between ServerMessage and RemoteService? From the lib directory, I guess RemoteService is specific to Android?